Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM golang:1.14.4-alpine3.12 AS BUILD
+
+RUN apk add make build-base
+
+WORKDIR /vegeta
+
+# cache dependencies
+ADD go.mod /vegeta
+ADD go.sum /vegeta
+RUN go mod download
+
+#now build source code
+ADD / /vegeta
+
+RUN go test -v ./...
+RUN make vegeta
+# RUN go build -v -o /bin/vegeta
+
+FROM alpine:3.12.0
+
+ENV TARGET_URL ''
+ENV DURATION '5'
+ENV REQUESTS_PER_SECOND '5'
```

Comment:
Rather than using environment variables, which would require us to map each flag to an env var, can we just expose the vegeta binary as a command so that we're able to pass flags directly to `docker run` ?

Reply:
done